standard_releases.md

```diff
@@ -1,59 +1,206 @@
+preferred over "v0-1" because someone might interpret the latter as the latest
+version of v0.1.
+
+## Translations and releases
+
+Our standard assumption for translations is that they are generally
+**asynchronous with development**. That means that changes to the standard or
+documentation do not need to wait for translation before they can be released,
+they should be planned and scheduled as appropriate. For small changes, this
+most likely means batching and performing translations when a suitable number
+are required. For larger changes, particularly new releases, we're more likely
+to want to have all supported translations in place before releasing.
+
+## Release processes
+
+How exactly we perform a release depends on what is being released and which
+versions the changes affect. The sections below detail specific scenarios, but
+the following general questions always apply:
+
+1. Has the schema or any normative documentation changed?
+2. Does text need translating?
+3. Do the changes affect any existing, released versions?
+
+### A new release
+
+Preparing a new release is the most straightforward scenario, though not the
+least work! The general process is:
+
+* Decide on the set of changes that will be released. Some of these will have
+  likely already been merged in the course of normal development, but they may
+  need additional development, examples adding, etc.
+* Review, test and merge those changes into the `master` branch.
+* Depending on the other ongoing work, either call a "freeze" on the master
+  branch, whereafter merging changes that are not going to be part of the
+  release cannot be merged, or create a new release branch from master. A
+  release freeze is preferred, to reduce the complexity of incorporating any
+  release fixes back into master, but this can be weighed against the impediment
+  to other development on a case-by-case basis.
+* Begin translation of the new release (see [translations](/translations)) and
+  fixing of any snags or bugs. If `master` is frozen, these changes can be merged
+  into master, if using a release branch, merge them there for incorporation
+  into master after the release.
+* Once the release is finalised and everything is ready, release the new version
+  on ReadTheDocs, either by making a new release branch, or simply enabling the
+  existing one. Remember to update the 'latest' branch pointer, and update other
+  release branches to point to it.
+* If fixes and snagging were done on a release branch, now is the time to merge
+  that branch back into master and fix any conflicts with what's been
+  merged in the meantime.
+
+### A new language to the current release
+
+Until such time that we support several stable releases, our assumption is that
+only the current release (and any future ones) will be translated into new
+languages.
+
+When there are no significant differences between `master` and the current
```

Comment:
When there are no significant differences between `master` and the current release, the process should be to use the existing project in Transifex. For example, BODS v0-2-0 has a project called "BODS v0.2" in Transifex. New languages for translation can be added at any time through the Transifex UI.
(I can add some screenshots to illustrate this)